h2o-genmodel/src/main/java/hex/genmodel/ModelMojoReader.java

```diff
@@ -56,7 +58,8 @@ public static MojoModel readFrom(MojoReaderBackend reader, final boolean readMod
       ModelMojoReader mmr = ModelMojoFactory.INSTANCE.getMojoReader(algo);
       mmr._lkv = info;
       mmr._reader = reader;
-      mmr.readAll(readModelDescriptor);
+      boolean isTargetEncodingUsed = info.containsKey("target_encoding_is_used") && ((RawValue) info.get("target_encoding_is_used")).parse(false);
```

Comment:
@deil87 can we avoid this? The TEMojoReader should be responsible for reading all supplementary information.

Can you move it to the TEReader or is there a reason why it cannot be there?

Reply:
@michalkurka Thanks! There is no need to move it. It should be deleted. It is an artefact from first version of Mojo support where I was going to store TE related stuff in the Model. There is now even writing counterpart path for this key. I will go through all the changes when I'm done with all design modifications.

Reply:
Perfect, thank you!